### DISCUSSION TITLE
Ramps1.6 configurations

### DISCUSSION THREAD

**Discussion by armature81** [arkypita/LaserGRBL#1084] [Q&A]
I just got my new ramps1.6 control board as i need to configure it for my laser machine
I need help to configure the follow.

Machine laser mode
M8&M7 out pin and configuration
5V PWM/TTL pin out to control laser power
12V pin out for laser drive board ( i looking to configure D8,D9 or D10)

**Comment by gmmanonymus111** [comment]
Since you are using RAMPS, with Arduino Mega (I suppose), I suggest take a look [here](https://github.com/fra589/grbl-Mega-5X)
There are a plenty of documentation/wiring diagram to help you get started with your custom engraver!
When you are finished, you might need [this tweak](https://github.com/gnea/grbl-Mega/issues/55#issuecomment-403261212) to enable 5v PWM signal
